plugins/cost-insights/src/components/LabelDataflowInstructionsPage/LabelDataflowInstructionsPage.tsx

```diff
@@ -79,11 +89,21 @@ sc.optionsAs[DataflowPipelineOptions].setLabels(Map("job-id" -> "my-dataflow-job
         </Typography>
         <Typography paragraph>
           For more information on specifying options, see the{' '}
-          <a href="https://cloud.google.com/dataflow/docs/guides/specifying-exec-params">
+          <a
+            {...coreHyperlinkAttr({
+              href:
+                'https://cloud.google.com/dataflow/docs/guides/specifying-exec-params',
+            })}
+          >
```

Comment:
Umm, I am sorry but this is not easy to understand and is unnecessary obfuscation. I would prefer `<a href="x" rel="y" target="z"></a>` instead.

Reply:
In that case we have to skip dynamic href's. For example

```jsx
<a href={variable} rel="?" />
```

So should I update to your need's? Or enhance naming convention which is more relevant?

Reply:
In my opinion, let's add `rel="noopener noreferrer"` to the one case identified by GitHub scan in #2760. And as a bonus fixing view `Link` components where the website is explicitly external would be great. I am not sure what to do with the variable href's - it will really be case by case. We can leave them for now.

I also appreciate your efforts so far and so thank you. :)

Reply:
Done! Please ping me whenever there is requirements that is relevant to my previous approach. Thanks